### PR TITLE
remove metrics style guide

### PR DESCRIPTION
**Pull request by Jstein77**
This PR removes the outdated metrics style guide.

### CODE REVIEW

**Comment by jthandy** [comment]
@Jstein77 assuming we're replacing this with something?

**Comment by Jstein77** [comment]
@jthandy Yes! Winnne and I are working on an updated set up best practices and style guide. In terms of sequencing, I think it would be best to remove this page so we don't have materials referencing the old metrics spec. We should have something to replace it with by the end of the week.

File: dbt_style_guide.md [hunk 1]
```diff
@@ -531,111 +531,156 @@ when needed.
   ) %}
   ```
 
-## Metrics style guide
+## Semantic Models & Metrics style guide
+
+## Organizing Semantic Model
```
Comment (dbeatty10) [inline dbt_style_guide.md:536]: Pluralize?
```suggestion
## Organizing Semantic Models
```

File: dbt_style_guide.md [hunk 2]
```diff
@@ -531,111 +531,156 @@ when needed.
   ) %}
   ```
 
-## Metrics style guide
+## Semantic Models & Metrics style guide
+
+## Organizing Semantic Model
+
+- Semantic models are created in the same yml file where you define configs, tests and metrics for your models. There should be one yml file per model.
```
Comment (dbeatty10) [inline dbt_style_guide.md:538]: Use the same capitalization of YAML that appears throughout the rest of the file:

```suggestion
- Semantic models are created in the same YAML file where you define configs, tests and metrics for your models. There should be one YAML file per model.

```

File: dbt_style_guide.md [hunk 3]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+├── dbt_project.yml
+└── models
+    ├── marts
+				├── finance
+					  ├── orders.sql
+						├── orders.yml #semantic models defined here
+    ├── staging
+			  ├── product
+				    ├── products.sql
+						├── products.yml #semantic models defined here
```
Comment (dbeatty10) [inline dbt_style_guide.md:551]: Similar to the [Python style guide for comments in PEP-8](https://peps.python.org/pep-0008/#inline-comments), inline comments in YAML should be separated by at least two spaces from the statement. They should start with a # and a single space.

```suggestion
						├── orders.yml  # semantic models defined here
    ├── staging
			  ├── product
				    ├── products.sql
						├── products.yml  # semantic models defined here

```

File: dbt_style_guide.md [hunk 4]
```diff
@@ -531,111 +531,156 @@ when needed.
...
-      ├── staging
-      └── metrics
-          ├── projects
-          |   ├── active_projects.yml
-          ├── accounts
-          |   ├── active_cloud_accounts.yml
-          └── users
-              ├── base__net_promoter_score.yml
-              └── net_promoter_score.yml
+- Metrics are created in the same yml file where you define configs, tests and semantic models for your models.
```
Comment (dbeatty10) [inline dbt_style_guide.md:556]:
```suggestion
- Metrics are created in the same YAML file where you define configs, tests and semantic models for your models.

```

File: dbt_style_guide.md [hunk 5]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+├── dbt_project.yml
+└── models
+    ├── marts
+				├── finance
+					  ├── orders.sql
+						├── orders.yml #metrics defined here
+    ├── staging
+			  ├── product
+				    ├── products.sql
+						├── products.yml  #metrics defined here
```
Comment (dbeatty10) [inline dbt_style_guide.md:569]:
```suggestion
						├── orders.yml  # metrics defined here
    ├── staging
			  ├── product
				    ├── products.sql
						├── products.yml  # metrics defined here

```

File: dbt_style_guide.md [hunk 6]
```diff
@@ -531,111 +531,156 @@ when needed.
...
-
-    calculation_method: count
-    expression: unique_id
-
-    timestamp: created_at
-    time_grains: [day, month, quarter, year]
-
+--orders.yml
+semantic_models:
+  #The name of the semantic model.
```
Comment (dbeatty10) [inline dbt_style_guide.md:577]:
```suggestion
  # The name of the semantic model.
```

File: dbt_style_guide.md [hunk 7]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+  - name: orders
+    defaults:
+      agg_time_dimension: ordered_at
+    description: |
+      Order fact table. This table is at the order grain with one row per order. 
+    #The name of the dbt model and schema
```
Comment (dbeatty10) [inline dbt_style_guide.md:583]:
```suggestion
    # The name of the dbt model and schema
```

File: dbt_style_guide.md [hunk 8]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+    model: ref('orders')
+    #Entities. These usually corespond to keys in the table.
```
Comment (dbeatty10) [inline dbt_style_guide.md:585]:
```suggestion
    # Entities. These usually corespond to keys in the table.
```

File: dbt_style_guide.md [hunk 9]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+    entities:
+      - name: order_id
+        type: primary
+      - name: product
+        type: foreign
+        expr: location_id
+  #Dimensions. Either categorical or time. These add additonal context to metrics. The typical querying pattern is Metric by Dimension.
```
Comment (dbeatty10) [inline dbt_style_guide.md:592]:
```suggestion
  # Dimensions. Either categorical or time. These add additonal context to metrics. The typical querying pattern is Metric by Dimension.  
```

File: dbt_style_guide.md [hunk 10]
```diff
@@ -531,111 +531,156 @@ when needed.
...
```
Comment (dbeatty10) [inline dbt_style_guide.md:639]: Fix misspelling and comment spacing:
```suggestion
  # Dimensions. Either categorical or time. These add additional context to metrics. The typical querying pattern is Metric by Dimension.  
```

File: dbt_style_guide.md [hunk 11]
```diff
@@ -531,111 +531,156 @@ when needed.
...
+	- name: order_gross_profit
+	    description: Gross profit from each order.
+	    type: derived
+	    label: Order Gross Profit
+	    type_params:
+	      expr: revenue - cost
+	      metrics:
+	        - name: revenue
+	        - name: order_cost
+	          alias: cost
```
Comment (dbeatty10) [inline dbt_style_guide.md:685]: Replacing tabs with spaces:
```suggestion
        - name: order_gross_profit
            description: Gross profit from each order.
            type: derived
            label: Order Gross Profit
            type_params:
              expr: revenue - cost
              metrics:
                - name: revenue
                - name: order_cost
                  alias: cost
```

**Review by dbeatty10 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**